Update macos-mdm-setup.md (#23081)
fleetdm/fleet: articles/macos-mdm-setup.md

To turn on Windows MDM features, head to this [Windows MDM setup article](https://fleetdm.com/guides/windows-mdm-setup).

## Turn on Apple MDM

Apple uses APNs to authenticate and manage interactions between Fleet and hosts.

To connect Fleet to APNs or renew APNs, head to the **Settings > Integrations > Mobile device management (MDM)** page. 

Then click **Turn on** under the Apple (macOS, iOS, iPadOS) MDM section.

> Apple requires that APNs certificates are renewed annually. 
> - If your certificate expires, you will have to turn MDM off and back on for all macOS hosts.
> - Be sure to use the same Apple ID from year-to-year. If you don't, you will have to turn MDM off and back on for all macOS hosts.

> ## Automatic Enrollment

Reviewer: ```suggestion
## Automatic enrollment
```

Hosts that automatically enroll will be assigned to a default team. You can configure the default team for macOS, iOS, and iPadOS hosts by:

> 1. If you have not already, create teams following [this guide](https://fleetdm.com/guides/teams#basic-article). Our [best practice](#best-practice) recommendation is to have a team for each device type.

Reviewer: ```suggestion
1. Creating teams, if you have not already, following [this guide](https://fleetdm.com/guides/teams#basic-article). Our [best practice](#best-practice) recommendation is to have a team for each device type.
```

2. Navigating to the **Settings > Integrations > Mobile device management (MDM)** page and clicking "Edit" under "Automatic enrollment".

> 3. Click on the "Actions" dropdown for the ABM token you want to update, and then click "Edit teams".

Reviewer: ```suggestion
3. Clicking on the "Actions" dropdown for the ABM token you want to update, and then clicking "Edit teams".
```

> 4. Use the dropdowns in the modal to select the default team for each type of host, and click "Save" to save your selections.

Reviewer: ```suggestion
4. Using the dropdowns in the modal to select the default team for each type of host, and clicking "Save" to save your selections.
```
